message_parser/src/test/java/cumulus_message_adapter/message_parser/AdapterUtilities.java

```diff
@@ -58,6 +58,10 @@ private static String getJsonResponse(String url) throws IOException {
         conn.setRequestProperty("Accept", "application/json");
         conn.setRequestProperty("User-Agent", "@cumulus/deployment");
 
+        if (System.getenv("GITHUB_TOKEN") != null) {
```

Comment:
This is setting the Auth field in the header instead of using the URL param?    👍🏻

Reply:
Yeah, specifying the token via query param is no longer supported:

https://developer.github.com/changes/2020-02-10-deprecating-auth-through-query-param/